### PR TITLE
Meta/validators private forest

### LINKED ISSUE #1562: Table 4a Private ownership subcategories validator
OJ: Add consistency check that sub categories of Private ownership (table 4a) should add up to the area of private ownership, if all subcategories have been reported on (Örjan)

### PR DESCRIPTION
**Pull request by sorja**
Resolves #1562 

- [x] Add validation to Private forest
- [x] Remove RowValidation
- [x] Add elapsed seconds in migrate-fra
- [x] Solve issue 1a, 1b table header is link

https://user-images.githubusercontent.com/5508251/194028128-f5220d33-6372-4666-9320-c9a89cab7568.mov

### CODE REVIEW

File: src/client/pages/AssessmentSection/DataTable/Table/Row/Row.ts [hunk 1]
```diff
@@ -2,33 +2,20 @@ import React from 'react'
...
   const { type } = row.props
 
-  // TODO: validation
-  if ([RowType.validationMessages].includes(type)) return null
-
-  // validation error rows are hidden in public view
-  if (type === RowType.validationMessages && !user) {
-    return null
-  }
-
```
Comment (sorja) [inline src/client/pages/AssessmentSection/DataTable/Table/Row/Row.ts:31]: Only removes unused component `RowValidation`

File: src/client/pages/AssessmentSection/DataTable/Table/Table.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ const Table: React.FC<Props> = (props) => {
                   const { colSpan, rowSpan } = Cols.getStyle({ cycle, col })
                   const columnName = headers[colIndex]
 
-                  let isOdpHeader = showODP && table.props.odp && odpYears?.includes(columnName)
+                  let isOdpHeader = showODP && table.props.odp && !col.props.labels && odpYears?.includes(columnName)
```
Comment (sorja) [inline src/client/pages/AssessmentSection/DataTable/Table/Table.tsx:65]: - [x] Solve issue 1a, 1b table header is link

File: src/i18n/resources/en.js [hunk 3]
```diff
@@ -1018,6 +1018,7 @@ The FRA team
     mustBeEqualToTotalGrowingStock: 'Value should be equal to Total Growing Stock (2a)',
     valuesAreInconsistentWithNetChange: 'Values are inconsistent with Forest area net change',
     valuesAreInconsistent1aOr1b: 'Values are inconsistent with Areas reported in tables 1a or 1b',
+    mustBeEqualToPrivateForest: 'Subcategory exceeds Private ownership',
```
Comment (sorja) [inline src/i18n/resources/en.js]: Added to i18n issue

**Comment by minotogna** [comment]
new validator message should say: 'Subcategories sum should be equal to Private Ownership'. and the error should be visible only on subcategories

File: src/client/pages/AssessmentSection/DataTable/Table/Table.tsx (hunk 2 above)
Comment (rfontanarosa) [reply src/client/pages/AssessmentSection/DataTable/Table/Table.tsx:65]: I like this!

**Review by minotogna — APPROVED** [review]

**Review by rfontanarosa — APPROVED** [review]

**Pull request merged**